app/lib/frontend/templates/listing.dart

```diff
@@ -121,3 +123,20 @@ class PageLinks {
     return min(fromSymmetry, max(currentPage!, fromCount));
   }
 }
+
+d.Node? _nameMatches(List<String>? matches) {
+  if (matches == null || matches.isEmpty) {
+    return null;
+  }
+  final singular = matches.length == 1;
+  final namePluralized = singular ? 'name' : 'names';
+  return d.p(children: [
+    d.b(text: 'Matching package $namePluralized: '),
```

Comment:
Could we detect if the match is exact, and in that case write 'Exact package name match:'

Reply:
Updated.